analytics_data_api/v0/views/learners.py

```diff
@@ -85,55 +90,63 @@ class LearnerListView(CourseViewMixin, generics.ListAPIView):
+            * segments: Classification, based on engagement, of each learner's
+              work in this course (for example, "highly_engaged" or
+              "struggling").
             * engagements: Summary of engagement events for a time span.
-                * videos_viewed: Number of times a video was played.
-                * problems_completed: Unique number of problems completed.
-                * problems_attempted: Unique number of problems attempted.
-                * discussions_contributed: Number of discussions (e.g. forum posts).
+                * videos_viewed: Number of times any course video was played.
+                * problems_completed: Number of unique problems the learner
+                  answered correctly.
+                * problems_attempted: Number of unique problems attempted.
+                  This is a count of the different problems the learner tried.
+                * discussions_contributed: Number of posts, responses, or
+                  comments the learner contributed to course discussions.
 
     **Parameters**
 
-        You can filter the list of learners by course ID and other parameters
-        such as enrollment mode and text search.  You can also control the
-        page size and page number of the response, as well as sort the learners
-        in the response.
-
-        course_id -- The course within which user data is requested.
-        page -- The page of results which should be returned.
-        page_size -- The maximum number of results which should be returned per page.
-        text_search -- A string to search over the name, username, and email of learners.
-        segments -- A comma-separated string of segments to which
-            learners should belong.  Semgents are "OR"-ed together.
-            Cannot use in combination with `ignore_segments`
-            argument.
-        ignore_segments -- A comma-separated string of segments to
-            which learners should NOT belong.  Semgents are "OR"-ed
-            together.  Cannot use in combination with `segments`
+        You can filter the list of learners by course ID and by other
+        parameters, including enrollment mode and text search. You can also
+        control the page size and page number of the response, as well as sort
+        the learners in the response.
+
+        course_id -- The course for which user data is requested.
+        page -- The page of results that should be returned.
+        page_size -- The maximum number of results to return per page.
+        text_search -- An alphanumeric string that is used to search name,
+            username, and email values to find learners.
+        segments -- A comma-separated string of segment names that is used
+            to select only those learners who are categorized in those
+            segments. Segments are connected by OR statements. Cannot be
```

Comment:
Learners who are categorized in *at least* one of the segments will be returned.

I agree with leaving out the bit about "OR statement" since it implies that the user of the API has to write their own boolean logic statements.